### PR TITLE
+faraday.0.8.0

### PR DESCRIPTION
**Pull request by seliopou**

### CODE REVIEW

File: packages/faraday/faraday.0.8.0/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+maintainer: "Spiros Eliopoulos <spiros@inhabitedtype.com>"
+authors: [ "Spiros Eliopoulos <spiros@inhabitedtype.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/inhabitedtype/faraday"
+bug-reports: "https://github.com/inhabitedtype/faraday/issues"
+dev-repo: "git+https://github.com/inhabitedtype/faraday.git"
+build: [
+  ["dune" "subst"] {dev}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
[the remark this answers is not in the document]
Comment (seliopou) [reply packages/faraday/faraday.0.8.0/opam:11]: Hmm I do test against 4.03 so this is surprising. Looking at the ci status, it looks like a lot of tests are frozen? Can you point me to a particular failure so I can fix the failures for another release?

[the remark this answers is not in the document]
Comment (seliopou) [reply packages/faraday/faraday.0.8.0/opam:11]: Nevermind I see the failure now.

[the remark this answers is not in the document]
Comment (mseri) [reply packages/faraday/faraday.0.8.0/opam:11]:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test & ocaml:version >= "4.05.0" }
```
I think you need to use a slightly different syntax for the bound

**Comment by seliopou** [comment]
Whoops, didn't mean to close this with the faraday PR, but this is being subsumed by #19062 anyways.

**Pull request closed without merging**